hub/gads-ui/src/components/Admin/GlobalSettings/GlobalSettings.js

```diff
@@ -0,0 +1,158 @@
+                                    label="JPEG Quality"
+                                    select
+                                    value={jpegQuality}
+                                    onChange={(e) => setJpegQuality(e.target.value)}
+                                    variant="outlined"
+                                >
+                                    {jpegQualityOptions.map((option) => (
+                                        <MenuItem key={option} value={option}>
+                                            {option}
+                                        </MenuItem>
+                                    ))}
+                                </TextField>
+                                <TextField
+                                    label="Scaling Factor (Android)"
+                                    select
+                                    value={scalingFactorAndroid}
+                                    onChange={(e) => setScalingFactorAndroid(e.target.value)}
+                                    variant="outlined"
+                                >
+                                    {scalingFactorOptionsAndroid.map((option) => (
+                                        <MenuItem key={option} value={option}>
+                                            {option}%
+                                        </MenuItem>
+                                    ))}
+                                </TextField>
+                                <TextField
+                                    label="Scaling Factor (iOS)"
+                                    select
+                                    value={scalingFactoriOS}
+                                    onChange={(e) => setScalingFactoriOS(e.target.value)}
+                                    variant="outlined"
+                                >
+                                    {scalingFactorOptionsiOS.map((option) => (
+                                        <MenuItem key={option} value={option}>
+                                            {option}%
+                                        </MenuItem>
+                                    ))}
+                                </TextField>
+                                <Button
+                                    variant="contained"
+                                    onClick={handleSave}
+                                    disabled={loading}
+                                    style={{
+                                        backgroundColor: '#2f3b26',
+                                        color: '#f4e6cd',
+                                        fontWeight: 'bold'
+                                    }}
+                                >
+                                    {loading ? 'Saving...' : 'Save Settings'}
+                                </Button>
+                                {status === 'success' && <Typography color="green">Settings saved successfully!</Typography>}
```

Comment:
Would you mind using the global snackbar like [here](https://github.com/shamanec/GADS/blob/3db8d6d9e2ba976048758b611d39c74c80c85b4d/hub/gads-ui/src/components/DeviceControl/StreamCanvas/StreamSettings.js#L60). I am trying to move to similar feedback across the app

Reply:
Of course. A snackbar offers a much better experience to the user. I will adjust that.

Reply:
Done ([commit](https://github.com/shamanec/GADS/pull/145/commits/f32160baa0d95c708e21c0c6d566171699a8c6c2))